x-pack/plugins/ml/public/application/trained_models/models_management/expanded_row.tsx

```diff
@@ -11,6 +11,7 @@ import {
   EuiBadge,
   EuiCodeBlock,
   EuiDescriptionList,
+  EuiDescriptionListProps,
```

Comment:
It looks like the switch to Emotion results in an increase to the font size for the `EuiDescriptionList` items - on this page going from `14px` to `16px`, so the items are now the same (larger) size as the section headings.

Should we be looking to edit the size back down to the previous, smaller size? We use the `EuiDescriptionList` in  around 15 places in ML.

<img width="1394" alt="image" src="https://user-images.githubusercontent.com/7405507/186207511-038efa30-5c00-4643-b20a-242fd38af349.png">

Reply:
Found that the `compressed` styles did, in fact, change. Will address in EUI.

Reply:
Resolved as a part of [e088ee6](https://github.com/elastic/kibana/pull/138837/commits/e088ee62f6fa0c43ea87c0f81739575eb65dc95d)